Assets/MRTK/Core/Extensions/UnityObjectExtensions.cs

```diff
@@ -51,12 +51,10 @@ public static void DestroyObject(Object obj, float t = 0.0f)
         }
 
         /// <summary>
-        /// Tests if the Unity object is null. Checks both the managed object and the underly Unity-managed native object
+        /// Tests if an interface is null, taking potential UnityEngine.Object derived class implementers into account
+        /// which require their overridden operators to be called
         /// </summary>
         /// <returns>True if either the managed or native object is null, false otherwise</returns>
-        public static bool IsNull(Object obj)
-        {
-            return obj == null || obj.Equals(null);
-        }
+        public static bool IsNull<T>(T obj) where T : class => obj == null || obj.Equals(null);
```

Comment:
This is being added, but is it used yet anywhere?

Reply:
Here are two places this would be usable:
![image](https://user-images.githubusercontent.com/32508825/80672333-b3933b00-8aac-11ea-845b-069d77657993.png)